android/src/main/java/com/tailscale/ipn/ui/viewModel/ExitNodePickerViewModel.kt

```diff
@@ -62,7 +64,11 @@ class ExitNodePickerViewModel(private val nav: ExitNodePickerNav) : IpnViewModel
           R.string.allow_lan_access,
           SettingType.SWITCH,
           isOn = MutableStateFlow(Notifier.prefs.value?.ExitNodeAllowLANAccess),
-          enabled = MutableStateFlow(true),
+          enabled =
+              MutableStateFlow(
+                  IpnViewModel.mdmSettings.value.get(
```

Comment:
I don't think this will do what you want. If the value of mdmSettings changes, this stateflow will not update. I think you want to use combine, like [here](https://github.com/tailscale/tailscale-android/blob/cf6a203f7ae9faf154541fed036ceec05e6e0b62/android/src/main/java/com/tailscale/ipn/ui/viewModel/DNSSettingsViewModel.kt#L51).